Update translation requests contact (#18941)
huggingface/transformers: docs/TRANSLATING.md

Once you have translated the `_toctree.yml` file, you can start translating the [MDX](https://mdxjs.com/) files associated with your docs chapter.

> > 🙋 If you'd like others to help you with the translation, you can either [open an issue](https://github.com/huggingface/transformers/issues) or tag @[GuggerSylvain](https://twitter.com/GuggerSylvain)
>  on Twitter to gain some visibility.

Reviewer: ```suggestion
> 🙋 If you'd like others to help you with the translation, you should [open an issue](https://github.com/huggingface/transformers/issues) and tag @sgugger.
```
